setup.py

```diff
@@ -26,11 +26,13 @@ def read(fname):
     license="MIT",
     keywords="gpu optimizers optimization 8-bit quantization compression",
     url="https://github.com/TimDettmers/bitsandbytes",
-    install_requires=['scipy'],
     packages=find_packages(),
     package_data={"": libs},
-    install_requires=['torch', 'numpy', 'scipy'],
-    extras_require={'benchmark': ['pandas', 'matplotlib']},
+    install_requires=['torch', 'numpy'],
+    extras_require={
+        'benchmark': ['pandas', 'matplotlib'],
```

Comment:
Are these runtime dependencies at all? The seem to be used only in scripts and are not part of the wheel? Maybe the `benchmark` extra is not needed at all?

Reply:
The idea is that when running the benchmark scripts, you'd do 

```
pip install -e .[benchmark]
```
to get the regular requirements plus the things required for benchmarking, or if you want that and the test deps,
```
pip install -e .[benchmark,test]
```

Reply:
Doesn’t this also make it part of the public metadata, e.g “pip install bitsandbytes[benchmark]” ?

Reply:
Yes, it does.